src/main/scala/uk/gov/hmrc/play/microservice/bootstrap/MicroserviceGlobal.scala

```diff
@@ -35,15 +34,15 @@ trait MicroserviceFilters {
 
   def metricsFilter: MetricsFilter = MetricsFilter
   
-  def authFilter: MicroserviceAuthorisationFilter
+  def authFilter: Option[EssentialFilter]
 
   protected lazy val defaultMicroserviceFilters: Seq[EssentialFilter] = Seq(
-    metricsFilter,
-    microserviceAuditFilter,    
-    loggingFilter,
+    Some(metricsFilter),
```

Comment:
they are only options in building this val. Otherwise they are not. This seems like an easy way to build a sequence with or without that one element without resorting to mutable sequence of a bunch of ifs